ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import Box from '../../../ui/box';
+import Typography from '../../../ui/typography';
+import CheckBox from '../../../ui/check-box';
+import I18nValue from '../../../ui/i18n-value';
+import {
+  COLORS,
+  DISPLAY,
+  FLEX_DIRECTION,
+  TYPOGRAPHY,
+} from '../../../../helpers/constants/design-system';
+import { getAdvancedGasFeeValues } from '../../../../selectors';
+import { setAdvancedGasFee } from '../../../../store/actions';
+
+import { useAdvancedGasFeePopoverContext } from '../context';
+
+const AdvancedGasFeeDefaults = () => {
+  const dispatch = useDispatch();
+
+  const {
+    baseFeeMultiplier,
+    maxPriorityFeePerGas,
+  } = useAdvancedGasFeePopoverContext();
+  const advancedGasFeeValues = useSelector(getAdvancedGasFeeValues);
+
+  const updateDefaultSettings = (value) => {
+    if (value) {
+      dispatch(
+        setAdvancedGasFee({
+          maxBaseFee: baseFeeMultiplier,
+          priorityFee: maxPriorityFeePerGas,
+        }),
+      );
+    } else {
+      dispatch(setAdvancedGasFee(null));
+    }
+  };
+  const defaultPreference =
+    Boolean(advancedGasFeeValues) &&
+    advancedGasFeeValues.maxBaseFee === baseFeeMultiplier &&
+    advancedGasFeeValues.priorityFee === maxPriorityFeePerGas;
+
+  return (
+    <Box
+      display={DISPLAY.FLEX}
+      flexDirection={FLEX_DIRECTION.ROW}
+      marginRight={4}
+      className="advanced-gas-fee-defaults"
+    >
+      <CheckBox
+        checked={defaultPreference}
+        className="advanced-gas-fee-defaults__checkbox"
+        onClick={() => updateDefaultSettings(!defaultPreference)}
+      />
+      <Typography variant={TYPOGRAPHY.H7} color={COLORS.UI4}>
```

Comment:
Add margin 0 to the Typography component here so it aligns better. I think a default margin being added to the Typography component was a mistake. Going to look into how we can resolve this soon

```suggestion
      <Typography variant={TYPOGRAPHY.H7} color={COLORS.UI4} boxProps={{margin: 0}}>
```

Before
<img width="411" alt="Screen Shot 2021-12-07 at 2 52 50 PM" src="https://user-images.githubusercontent.com/8112138/144951569-52450932-f912-4805-a1b6-1bf5a6c4271e.png">

After
<img width="390" alt="Screen Shot 2021-12-07 at 2 50 38 PM" src="https://user-images.githubusercontent.com/8112138/144951485-2220d8b9-1c27-4e6f-8e37-8fb25a789927.png">